xtext.cpp

```diff
@@ -2063,9 +2063,8 @@ class XText {
         free(s_utf8);
     }
 
-    // Get (as a single UTF-8 string) the segment of m_text, extended to include
-    // the full words that may be cut at boundaries (start, end).
-    void getSelectedWords(int start, int end, int context) {
+    // Get extended start and end indices that include the full words that may be cut at boundaries (start, end).
```

Comment:
```suggestion
    // Get start and end indices of an extended segment of m_text that includes the full words that may be cut at boundaries (start, end).
```
(Just for clarity, "extended indices" reads odd to me.)

Reply:
I updated the comments.